src/phockup.py

```diff
@@ -111,8 +113,11 @@ def get_file_name(self, original_filename, date):
         Generate file name based on exif data unless it is missing or
         original filenames are required. Then use original file name
         """
+        local_path_filename, local_file_extension = os.path.splitext(original_filename)
```

Comment:
Can you rename the `local_path_filename` and `local_file_extension` to `filename` and `ext`